sql/ddl_internal.sql

```diff
@@ -1,23 +1,25 @@
 -- Creates a hypertable row.
 CREATE OR REPLACE FUNCTION _timescaledb_internal.create_hypertable_row(
-    schema_name             NAME,
-    table_name              NAME,
-    time_column_name        NAME,
-    time_column_type        REGTYPE,
-    partitioning_column     NAME,
-    number_partitions       INTEGER,
-    associated_schema_name  NAME,
-    associated_table_prefix NAME,
-    chunk_time_interval        BIGINT,
-    tablespace              NAME
+    schema_name              NAME,
+    table_name               NAME,
+    time_column_name         NAME,
+    time_column_type         REGTYPE,
+    partitioning_column      NAME,
+    partitioning_column_type REGTYPE,
+    number_partitions        INTEGER,
+    associated_schema_name   NAME,
+    associated_table_prefix  NAME,
+    chunk_time_interval      BIGINT,
+    tablespace               NAME
```

Comment:
So, this is not the public-facing API, which does not allow you to give a tablespace. The tablespace here is only non-NULL if the "main table" was created with `CREATE TABLE foo .... TABLESPACE bar...`, and then tablespace here will be set to bar `bar`.